pkg/tarsum/versioning.go

```diff
@@ -17,7 +17,7 @@ type Version int
 const (
 	Version0 Version = iota
 	Version1
-	// NOTE: this variable will be either the latest or an unsettled next-version of the TarSum calculation
+	// VersionDev (NOTE) this variable will be either the latest or an unsettled next-version of the TarSum calculation
```

Comment:
This won't look right in godoc, and the comment doesn't really make sense because it says "variable" but it's describing a constant.